store/tikv/coprocessor.go

```diff
@@ -367,6 +367,7 @@ type copIteratorWorker struct {
 	respChan chan<- *copResponse
 	finishCh <-chan struct{}
 	vars     *kv.Variables
+	clientHelper
```

Comment:
Is `clientHelper` going to be used in other places?

Reply:
I'm not sure, but I can export it when it's used.